### PR TITLE
ZATo2LA01j_5f_NLO_FXFX and WAToLNuA01j_5f_NLO_FXFX with 306000 centra…

### PR DESCRIPTION
**Pull request by junholeephy**
…l pdf

### CODE REVIEW

**Comment by junholeephy** [comment]
We need ZATo2LA01j_5f_NLO_FXFX for VBS ZGamma study, WAToLNuA01j_5f_NLO_FXFX for VBS WGamma study. The main changes of the cards are
(1) central pdf changed from 292200 to 306000, error PDF-sets changed correspondingly,
(2) ptl (Min lepton transverse momentum) changed from 0 to 15 GeV,
(3) ptgmin (Min photon transverse momentum) changed from 10 to 15 GeV,
(4) Default pdfset included due to systematics setting in "runcmsgrid_NLO.sh", totally 964 pdf weights involved for each event.

Since statistic of these samples have been a problem VBS ZA & WA study,  
(2),(3) is for consideration of luminosity of the samples with limited events number, since relatively large XSec.  With same event number of 2016MC's, we can earn doubled luminosity of each sample.

**Comment by junholeephy** [comment]
Also, We are aiming Moriond 18 for VBS ZA, LHCP 18 for VBS WA analysis. 
Hopefully, we can get high priority production of these samples.

**Comment by junholeephy** [comment]
I have tried to compare "LHE level" MG_26x with 2016MC Official WA_NLO_FXFX sample. (292200 pdf is picked as central-pdf for both MC, just for correspondence comparison.)
Result is here :: https://indico.cern.ch/event/671984/contributions/2748761/attachments/1539505/2414902/WA_MG_comparison.pdf

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model loop_sm-no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_proc_card.dat]: please remove unused definitions of l and vl

**Comment by perrozzi** [comment]
I am not sure why you put the datacards in the pre2017 folder if they are meant to be used for Moriond18 compatible results. Am I missing anything?

**Comment by junholeephy** [comment]
Thanks for your comments! 
Does moving these datacards to directory "MadGraph5_aMCatNLO/cards/production/13TeV" make sense for Moriond18 with 2016 Datasets?

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_proc_card.dat]: remove this banner as is outdated

**Comment by perrozzi** [comment]
if is to compare with 2016 data then pre2017 is correct, yes

**Comment by junholeephy** [comment]
The outdated banner and some definition on proc_card is modified.
Thank you.

**Comment by efeyazgan** [comment]
Please remove the commented input.

**Comment by junholeephy** [comment]
Just noticed the tune for 2016MC used CUETMP1 which exploits NNPDF30 NLO (292200),  so I think we should change the central PDF set from 306000 to 292200, as we plan to produce VGamma samples to compare with 2016 DATA.  What do you think?

**Comment by perrozzi** [comment]
I think that the central value of the 2 pdf sets is exactly the same, only the variations are differenc (replicas vs eigenvariations) :-)
292200 | NNPDF30_nlo_nf_5_pdfas 
306000 | NNPDF31_nnlo_hessian_pdfas
cfr https://lhapdf.hepforge.org/pdfsets.html

**Comment by perrozzi** [comment]
ah no, wait! you are right....

**Comment by junholeephy** [comment]
Thank you, I have created a new pull request "https://github.com/cms-sw/genproductions/pull/1445".
Please take a look.

**Comment by perrozzi** [comment]
sorry, why a new PR? You can just replace the PDF dataset in this PR, right?

**Comment by junholeephy** [comment]
Is it possible making another "commit"  while the PR is merged?
Thanks for your comment.

**Comment by perrozzi** [comment]
I had not noticed this was merged, re-opening the other one :-)

**Comment by junholeephy** [comment]
Thank you, the new PR is here "https://github.com/cms-sw/genproductions/pull/1455". Please take a look!

**Comment by junholeephy** [comment]
Considering discrepancy of photon_eta [1], I have put these on mg26x branch. Should I move it to the master branch?

[1] https://indico.cern.ch/event/663254/contributions/2710009/attachments/1518570/2371281/Report_ZG_20170905T.pdf

**Pull request merged**